sdk/src/test/java/io/customer/sdk/CustomerIOTest.kt

```diff
@@ -28,13 +29,14 @@ internal class CustomerIOTest : BaseTest() {
     }
 
     @Test
-    fun `verify SDK configuration is correct`() {
+    fun `verify SDK default configuration is correct`() {
         customerIO.config.siteId `should be equal to` MockCustomerIOBuilder.siteId
         customerIO.config.apiKey `should be equal to` MockCustomerIOBuilder.apiKey
         customerIO.config.timeout `should be equal to` MockCustomerIOBuilder.timeout.toLong()
         customerIO.config.region `should be equal to` MockCustomerIOBuilder.region
         customerIO.config.urlHandler `should be equal to` MockCustomerIOBuilder.urlHandler
         customerIO.config.autoTrackScreenViews `should be equal to` MockCustomerIOBuilder.shouldAutoRecordScreenViews
+        customerIO.config.autoTrackDeviceAttributes `should be equal to` MockCustomerIOBuilder.autoTrackDeviceAttributes
```

Comment:
Here is the false positive test. I believe that if you were to change `sdk/src/test/java/io/customer/sdk/MockCustomerIOBuilder.kt` default value of `autoTrackDeviceAttributes` to `false` this test function would fail.